### ISSUE TITLE
Links with bibliography

### ISSUE THREAD

**Issue by Ciflire** [nvarner/typst-lsp#433]
- **Component**: 
    - LSP inside helix
- **LSP version**: 0.12.1
- **OS version and name**: NixOS 24.05

[x] I am on the [latest](https://github.com/nvarner/typst-lsp/tags) stable version of the extension/LSP.
[x] I have searched the [issues](https://github.com/nvarner/typst-lsp/issues) of this repo and believe that this is not a duplicate.

## Issue

I have a multiple file typst project.
```
.
├── chapters
│   ├── 01-coverPage.typ
│   ├── 02-nonPlagiat.typ
│   ├── 03-contact.typ
│   ├── 04-foreword.typ
│   ├── 05-thanks.typ
│   ├── 06-outline.typ
│   ├── 07-introduction.typ
│   ├── 08-presentation.typ
│   ├── 09-tasks.typ
│   ├── 10_1-reflexion.typ
│   ├── 10_2-reflexion.typ
│   ├── 11-conclusion.typ
│   ├── 12-bibliography.bib
│   ├── 12-bibliography.typ
│   ├── 13-illustrations.typ
│   ├── 14-table.typ
│   ├── 15-listings.typ
│   └── 16-glossary.typ
├── figures
│   ├── anonymous_company-logo.svg
│   ├── collegium-logo.svg
│   ├── school-logo.svg
│   └── university-logo.svg
├── main.typ
├── README.md
└── template.typ
```
The lsp seems to provide the value of the link, it gives the autocompletion. But i'm also getting an error that the label does not exist.
![image](https://github.com/nvarner/typst-lsp/assets/39668077/4bd21b1e-0fa3-43ac-8e11-d53252611d9f)

It does work inside the document where i create the bibliography but nowhere else, either above or below that document

**Comment by Ciflire** [comment]
Closed as duplicate of #366

**Comment by Ciflire** [comment]
Closed as duplicate of #366
